packages/sync/src/class-users.php

```diff
@@ -44,13 +43,23 @@ class Users {
 	 * @todo Eventually, connection needs to be instantiated at the top level in the sync package.
 	 */
 	public static function init() {
-		self::$connection = new Jetpack_Connection();
+		add_action( 'jetpack_loaded', array( __CLASS__, 'on_jetpack_loaded' ) );
+	}
+
+	/**
+	 * Runs on Jetpack being ready to load its packages.
+	 *
+	 * @param Jetpack $jetpack object.
+	 */
+	public static function on_jetpack_loaded( $jetpack ) {
+		self::$connection = $jetpack->connection_manager;
```

Comment:
Maybe a better idea would be to have a getter method for the connection manager. That way, we could evenetually mock that method when testing.

Reply:
Yes, thanks for the suggestion, I'll make a method like this.